TerminalDocs/customize-settings/color-schemes.md

```diff
@@ -3,16 +3,44 @@ title: Windows Terminal Color Schemes
 description: Learn how to create color schemes for Windows Terminal.
 author: cinnamon-msft
 ms.author: cinnamon
-ms.date: 05/19/2020
+ms.date: 7/28/2020
 ms.topic: how-to
 ms.service: terminal
 ms.localizationpriority: high
 ---
 
 # Color schemes in Windows Terminal
 
+Windows Terminal lets you define your own color schemes, either by using the built-in preset themes, or by creating your own theme from scratch. To change  themes, you'll need to edit the settings.json file in an editor such as [Visual Studio Code](https://code.visualstudio.com/).
+
+## Switching to a different color scheme
+
+Launch the Windows Terminal, and then select the small downward-facing arrow in the title bar. This will open a pull-down menu that lists the available profiles on your system (for example, Windows PowerShell and Command Prompt) and some other options. Select **Settings**, and the settings.json file will open in your default text editor.
```

Comment:
```suggestion
Launch Windows Terminal and then select the small downward-facing arrow in the title bar. This will open a pull-down menu that lists the available profiles on your system (for example, Windows PowerShell and Command Prompt) and some other options. Select **Settings**, and the settings.json file will open in your default text editor.
```